launch/launch/main.py

```diff
@@ -35,14 +35,38 @@ def main(argv=sys.argv[1:]):
         type=file_exists,
         nargs='+',
         help='The launch file.')
+    parser.add_argument(
+        '--args',
+        metavar="arg",
+        type=str,
+        nargs='+',
+        help='An argument to the launch file (e.g., arg_name:=value). All ' +
```

Comment:
The trailing ` +` is not necessary. Python automatically concats string literals.